fixture/db/client.json

```diff
@@ -19,7 +19,7 @@
             "client_secret_expires_at": 0,
             "client_secret": "VVrUT7n82KNcuh947RAI-dqY79K8r47k_Xk9WvdKXl_viltV5kzX5oNDjen97vr0pBRSlv3oGZvz3_ka1136LQ",
             "redirect_uris": ["https://localhost:8082/signin-oidc"],
-            "scope": "openid pools:read pools:write withdrawals:read rewards:write promotions:read promotions:write deposits:write deposits:read"
+            "scope": "openid account:read pools:read pools:write erc20:read erc20:write erc721:read erc721:write rewards:read rewards:write withdrawals:read deposits:read deposits:write promotions:read promotions:write widgets:write widgets:read transactions:read payments:read payments:write"
```

Comment:
@GarfDev  Shouldn't the `brands:read` and `brands:write`scopes be whitelisted for the dashboard? Thats is where the resource is managed right?

I noticed they are added in the oidc request config over here:
https://github.com/thxprotocol/dashboard/pull/60/files